modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="installing-topology-aware-lifecycle-manager-using-cli_{context}"]
+= Installing the {cgu-operator-full} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {cgu-operator-first}.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of the {rh-rhacm} Operator.
+* Set up a hub cluster with disconnected registry.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `talm-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-topology-aware-lifecycle-manager-subscription
+  namespace: openshift-operators
+spec:
+  channel: "4.10"
+  name: topology-aware-lifecycle-manager
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f talm-subscription.yaml
+----
+
+.Verification
+
+. Verify that the installation succeeded by inspecting the CSV resource:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-operators
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE                                          NAME                                     DISPLAY                                      VERSION   REPLACES                             PHASE
+openshift-operators    cluster-group-upgrades-operator.v4.10.0 cluster-group-upgrades-operator              4.10.0                                          Succeeded
```

Comment:
I'm not sure about the csv name. Please confirm with @nishant-parekh 
It should be something like `toplogy-aware-lifecyle-<??>.<version>` in my view.

Reply:
csv name should remain the same.

Reply:
@yliu127 Sorry, I missed this comment. Updated now!